openqa-maintenance.py

```diff
@@ -23,7 +23,7 @@
 import sys
 from datetime import date
 import md5
-import json
+import simplejson as json
```

Comment:
Better make that an exception handled - as we for ElementTree in other places - simplejson has its name for a reason (I assume)